src/components/searchForn/searchForm.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+import './searchForm.css';
+const SearchForm=()=>{
+let [searchedInput,setSearchedInput]=useState("");
+const handleOnChange = (e) => {
+    setSearchedInput(e.target.value);
+};
+const onSearch=(e)=>{
+    e.preventDefault();
+    console.log(searchedInput)
```

Comment:
Please remove console.log.

Reply:
Kept it for now, to check the search action is working properly or not. Can be removed later when we actually implement the search api call. Thoughts?